cdci_data_analysis/pytest_fixtures.py

```diff
@@ -716,7 +716,43 @@ def create_scw_list_file(list_length, format='list', scw_list=None):
         return f'scw_list_files/{file_name}'
 
     @staticmethod
-    def create_catalog_file(catalog_value):
+    def create_catalog_object(long=False):
+        catalog_object_dict = dict(
+            cat_lon_name="ra",
+            cat_lat_name="dec",
+            cat_frame="fk5",
+            cat_coord_units="deg",
+            cat_column_list=[[1], ["Test A"], [6], [5], [4], [3], [2], [1], [0]],
+            cat_column_names=["meta_ID", "src_names", "significance", "ra", "dec", "NEW_SOURCE", "ISGRI_FLAG", "FLAG",
+                              "ERR_RAD"],
+            cat_column_descr=[['meta_ID', '<i8'], ['src_names', '<U20'], ['significance', '>f4'], ['ra', '>f4'],
+                              ['dec', '>f4'], ['NEW_SOURCE', '>i2'], ['ISGRI_FLAG', '<i8'], ['FLAG', '<i8'],
+                              ['ERR_RAD', '<f8']]
+        )
+
+        if long:
+            # generate a very long catalog
+            catalog_object_dict['cat_column_list'] = \
+                [[1, 2, 3, 4, 5, 6, 7, 8, 9],
+                 ['1E 1740.7-2942', '4U 1700-377', 'GRS 1758-258', 'GX 1+4', 'GX 354-0', 'GX 5-1', 'IGR J17252-3616',
+                  'SLX 1735-269', 'Swift J1658.2-4242'],
+                 [50.481285095214844, 29.631359100341797, 39.41709899902344, 19.39865493774414,
+                  17.236827850341797, 10.458189964294434, 7.3749494552612305, 8.645143508911133, 8.171965599060059],
+                 [265.97705078125, 255.96563720703125, 270.2925720214844, 263.0119934082031, 263.00067138671875,
+                  270.2991027832031, 261.33197021484375, 264.5558166503906, 254.55958557128906],
+                 [-29.746740341186523, -37.84686279296875, -25.736726760864258, -24.74085235595703, -33.82389831542969,
+                  -25.082794189453125, -36.24260330200195, -27.056262969970703, -42.71879196166992],
+                 [-32768, -32768, -32768, -32768, -32768, -32768, -32768, -32768, -32768],
+                 [2, 2, 2, 2, 2, 2, 1, 2, 2],
+                 [0, 0, 0, 0, 0, 0, 0, 0, 0],
+                 [2.9999999242136255e-05, 0.0002800000074785203, 0.0002800000074785203,
+                  0.0002800000074785203, 0.0002800000074785203, 0.0008299999753944576,
+                  0.0011099999537691474, 0.00016999999934341758, 5.555555617320351e-05]]
+
```

Comment:
not directly suggestion, but please use either this string directly, or use it as inspiration. Note absence of spaces. It's hard to break line without spaces, there are many edge cases. We could probably break by comas at least.

```suggestion
  selected_catalog"="{\"cat_column_descr\":[[\"meta_ID\",\"<i8\"],[\"src_names\",\"<U20\"],[\"significance\",\">f4\"],[\"ra\",\">f4\"],[\"dec\",\">f4\"],[\"NEW_SOURCE\",\">i2\"],[\"ISGRI_FLAG\",\"<i8\"],[\"FLAG\",\"<i8\"],[\"ERR_RAD\",\"<f8\"]],\"cat_column_list\":[[1,2,3,4,5,6,7,8,9],[\"1E 1740.7-2942\",\"4U 1700-377\",\"GRS 1758-258\",\"GX 1+4\",\"GX 354-0\",\"GX 5-1\",\"IGR J17252-3616\",\"SLX 1735-269\",\"Swift J1658.2-4242\"],[50.481285095214844,29.631359100341797,39.41709899902344,19.39865493774414,17.236827850341797,10.458189964294434,7.3749494552612305,8.645143508911133,8.171965599060059],[265.97705078125,255.96563720703125,270.2925720214844,263.0119934082031,263.00067138671875,270.2991027832031,261.33197021484375,264.5558166503906,254.55958557128906],[-29.746740341186523,-37.84686279296875,-25.736726760864258,-24.74085235595703,-33.82389831542969,-25.082794189453125,-36.24260330200195,-27.056262969970703,-42.71879196166992],[-32768,-32768,-32768,-32768,-32768,-32768,-32768,-32768,-32768],[2,2,2,2,2,2,1,2,2],[0,0,0,0,0,0,0,0,0],[0.000029999999242136255,0.0002800000074785203,0.0002800000074785203,0.0002800000074785203,0.0002800000074785203,0.0008299999753944576,0.0011099999537691474,0.00016999999934341758,0.00005555555617320351]],\"cat_column_names\":[\"meta_ID\",\"src_names\",\"significance\",\"ra\",\"dec\",\"NEW_SOURCE\",\"ISGRI_FLAG\",\"FLAG\",\"ERR_RAD\"],\"cat_coord_units\":\"deg\",\"cat_frame\":\"fk5\",\"cat_lat_name\":\"dec\",\"cat_lon_name\":\"ra\"}"
```